cluster/gce/gci/configure-helper.sh

```diff
@@ -577,6 +577,31 @@ rules:
       - group: "" # core
         resources: ["events"]
 
+  # node and pod status calls from nodes are high-volume and can be large, don't log responses for expected updates from nodes
+  - level: Request
+    users: ["kubelet", "system:node-problem-detector", "system:serviceaccount:kube-system:node-problem-detector"]
+    verbs: ["update","patch"]
+    resources:
+      - group: "" # core
+        resources: ["nodes", "nodes/status", "pods", "pods/status"]
```

Comment:
Why are `nodes` and `pods` included here? I'm OK with it being request level, but I'm curious what update/patch calls to those objects are frequent.

Reply:
rechecked output of e2e runs and you're right, only the status ones were high volume. update.